### PR TITLE
try codeclimate coverage on GIDS

### PR DESCRIPTION
**Pull request by annaswims**
Much like https://github.com/department-of-veterans-affairs/vets-api/pull/3235  introduce CodeClimate coverage reports to GIDS

### CODE REVIEW

**Review by jholton — APPROVED** [review]
:shipit:

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +19,9 @@ RUN yum install -y fontconfig freetype freetype-devel fontconfig-devel libstdc++
   tar -xjvf ./phantomjs-2.1.1-linux-x86_64.tar.bz2 --strip-components 1 -C /opt/phantomjs/ && \
   ln -s /opt/phantomjs/bin/phantomjs /usr/bin/phantomjs
 
+RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /cc-test-reporter
+RUN chmod +x /cc-test-reporter
+
```
Comment (kfrz) [inline Dockerfile]: I couldn't get the Dockerfile to build locally with `docker-compose up` on this branch, but it failed on the `groupadd -r gibct` command with code `139`  which seems odd

**Review by kfrz — APPROVED** [review]
:+1: Looks good to me.

File: Dockerfile (hunk 1 above)
Comment (annaswims) [reply Dockerfile]: It's likely unrelated to this branch.  https://github.com/department-of-veterans-affairs/gibct-data-service/pull/431 just changed the Dockerfile significantly.

**Pull request merged**